app/main.py

```diff
@@ -1,6 +1,22 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+KEY = os.getenv("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json"
+LOCATION = os.getenv("city")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    current_param = {"q": LOCATION, "key": KEY}
```

Comment:
The 'current_param' could be defined as a constant at the top of the file, after the 'CITY' and 'API_KEY' constants. This way, it would be easier to change the parameters if needed and the code would be cleaner.